app/components/GlobalSyncStatus.vue.ts

```diff
@@ -0,0 +1,27 @@
+import { Component } from 'vue-property-decorator';
+import Vue from 'vue';
+import { Inject } from 'util/injector';
+import { MediaBackupService, EGlobalSyncStatus } from 'services/media-backup';
+
+@Component({})
+export default class GlobalSyncStatus extends Vue {
+  @Inject() mediaBackupService: MediaBackupService;
+
+  get syncIcon() {
+    const status = this.mediaBackupService.globalSyncStatus;
+
+    return {
+      'fa-cloud': status === EGlobalSyncStatus.Synced,
+      'fa-refresh': status === EGlobalSyncStatus.Syncing,
+      'fa-spin': status === EGlobalSyncStatus.Syncing
```

Comment:
The last two comparsions is equal

Reply:
Yeah that was intentional.  The object is meant to map individual classes to boolean values for Vue to unroll into a class string.  It might be possible to combine multiple classes into one key, I'm not sure.